packages/central-server/src/apiV2/import/importSurveys/importSurveyQuestions.js

```diff
@@ -262,10 +279,15 @@ export async function importSurveysQuestions({ models, file, survey, dataGroup,
       await configImporter.add(rowIndex, componentId);
     } catch (error) {
       const validationError = constructImportValidationError(error.message, 'config');
-      throw validationError;
+      errors.push(validationError);
+      continue;
     }
   }
 
+  if (errors.length > 0) {
+    throw new MultiValidationError('Errors occurred while importing questions', errors);
+  }
```

Comment:
```suggestion
  if (errors.length > 0) {
    throw new MultiValidationError(
      'Errors occurred while importing questions',
      errors.map(({ message, extraFields }) => ({
        message,
        extraFields,
      })),
    );
  }
```
Assuming we've done the above comment to switch `errors` to be a list of `Error` instances